packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js

```diff
@@ -62,10 +62,10 @@ function setupRegularPresetTaskFactory(
     Full node                   - Host the full Dash blockchain (no collateral)
     Masternode                  - Full node features, plus Core services such as
                                   ChainLocks and InstantSend (1000 DASH collateral)
-    High-performance full node  - Full node features, plus host a full copy of the
-                                  Platform blockchain (no collateral)
-    High-performance masternode - Masternode features, plus Platform services such
-                                  as DAPI and Drive (4000 DASH collateral)\n`,
+    Evo full node  - Full node features, plus host a full copy of the Platform
+                                  blockchain (no collateral)
+    Evo masternode - Masternode features, plus Platform services such as DAPI and
+                                  Drive (4000 DASH collateral)\n`,
```

Comment:
Maybe we should spell out "Evolution" here? "Evo" is kinda an abbreviation.

Reply:
Sounds good to me